requirements.txt

```diff
@@ -39,7 +39,7 @@ ipython==8.2.0
 ipython-genutils==0.2.0
 jsonfield==3.1.0
 jsonschema==3.2.0
-kombu==5.0.2
+kombu
```

Comment:
@Lawful2002 You have pushed the changes of `requirements.txt`, kindly remove these from the PR.